internal/api/browser_extension/service/service.go

```diff
@@ -148,12 +148,11 @@ func (m *BrowserExtensionModule) RegisterPublicRoutes(router *gin.Engine) {
 	publicRouter.GET("/browser_extensions/:extension_id/2fa_requests/:token_request_id", m.RoutesHandler.GetBrowserExtension2FaTokenRequest)
 	publicRouter.POST("/browser_extensions/:extension_id/2fa_requests/:token_request_id/commands/close_2fa_request", m.RoutesHandler.Close2FaRequest)
 
-	if m.Config.IsTestingEnv() {
-		publicRouter.DELETE("/browser_extensions", m.RoutesHandler.RemoveAllBrowserExtensions)
-		publicRouter.DELETE("/browser_extensions/devices", m.RoutesHandler.RemoveAllBrowserExtensionsDevices)
-	}
-
 }
 func (m *BrowserExtensionModule) RegisterAdminRoutes(g *gin.RouterGroup) {
+	if m.Config.IsTestingEnv() {
```

Comment:
just to be consistent with other changes in #9 , all testing endpoints are under admin routes.